config/routes.rb

```diff
@@ -7,6 +7,10 @@
     namespace :v1 do
       resources :upload, only: [:create], controller: 'search_results'
 
+      devise_for :users, controllers: {
+        omniauth_callbacks: 'api/v1/users/omniauth_callbacks'
+       }
```

Comment:
```suggestion
      }
```

Reply:
🙈 The 'Format Document' doesn't work at all.